esp/src/Login.html

```diff
@@ -147,6 +150,10 @@
                         var element = document.getElementById('hidden_msg');
                         element.style.display = "inline-block";
                         deleteCookie('ESPAuthenticationMSG', '', 0);
+                    } else if (document.cookie.indexOf("ESPUserAcctError") > -1) {
+                        var element = document.getElementById('disbaled_msg');
```

Comment:
There's a typo here for the element's Id (disbaled...) Alternatively, you could use the `disabled` var from line 117 rather than finding it in the DOM again.